doc/changelog/09-coqide/11414-remove-ide-tactic-menu.rst

```diff
@@ -0,0 +1,4 @@
+- **Removed:**
+  Removed the "Tactic" menu from CoqIDE
```

Comment:
Just so that people better understand the reason behind this choice:

```suggestion
  Removed the "Tactic" menu from CoqIDE which had been unmaintained for a number of years
```